src/main/java/com/thecookiezen/infrastructure/docker/DockerFetcher.java

```diff
@@ -22,11 +21,15 @@
 
     @PostConstruct
     void init() {
-        DockerClientConfig config = DefaultDockerClientConfig.createDefaultConfigBuilder()
-                .withDockerHost("tcp://0.0.0.0:2376")
-                .withApiVersion("1.24")
+        dockerClient = getDockerClient("tcp://0.0.0.0:2375", "1.24");
```

Comment:
\[Checkstyle\] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4\.